### PR TITLE
Fix tagging user profiles and add JS ajax function

### LINKED ISSUE #6724: Center align the Pagination Bar (for additional pages of content)
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

This issue is created to align the navigation bar to center.
Screenshot (112)

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [$FILENAME]($BRANCH_URL) in the `$REPO` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
$DIFF
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by nstjean**
Fixes #6724  (<=== Add issue number here)
@jywarren This references what we talked about in #6422 - do I need to create an issue for this?

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

+++

In this PR:
- fixed the form to add tags to a user profile, which wasn't working
- added a test to ensure that adding tags via the existing javascript methods work
- added styling to all badge tags based on what was already done for some node tags - added a FA icon instead of an x
- also added a link to the tag badge so when it is clicked on it will go to the tag page

The badges for the node tags:
FireShot Capture 081 - 🎈 Public Lab_ Where can I find information on caves_ - localhost

Badges for the user profile tags:
FireShot Capture 082 - 🎈 Public Lab_ admin - localhost

### CODE REVIEW

File: test/system/tag_test.rb [hunk 1]
```diff
@@ -5,26 +5,47 @@
 class TagTest < ApplicationSystemTestCase
   Capybara.default_max_wait_time = 60
 
-  test 'adding a tag via javascript' do
-    visit '/'
+  # test 'adding a tag via javascript' do
```
Comment (jywarren) [inline test/system/tag_test.rb]: Just checking, did you want to comment all these out?

**Comment by jywarren** [comment]
And noting, I think this will have an effect on https://github.com/publiclab/plots2/issues/5882 - perhaps solving it? Or maybe 5882 can be solved by refactoring to use this method? 

Thanks @nstjean !!! Great work!

**Comment by nstjean** [comment]
Oops, no I didn't mean to leave those uncommented! Pushed up the fix.

Oh that would be great if it solved that! I'll check it out!

**Comment by jywarren** [comment]
Merging! Thanks -- you're killing it on the system tests! This is super!!!

**Comment by jywarren** [comment]
Aha, this issue may also be solved by your new code, though it may also need some tweaking now!

https://github.com/publiclab/plots2/issues/5882

**Comment by jywarren** [comment]
Oh sorry that is the exact same issue. Just doing some cleanup and... made it worse! Ignore!!!

**Pull request merged**